OpenRA.Game/Traits/SelectionDecorations.cs

```diff
@@ -78,6 +78,14 @@ IEnumerable<IRenderable> DrawPips(WorldRenderer wr, Actor self, int2 basePositio
 			if (!pipSources.Any())
 				yield break;
 
+			// Don't draw pips if this is the enemy vehicle
+			var isAlly = self.Owner.IsAlliedWith(self.World.LocalPlayer)
```

Comment:
This looks like it will break the ability to view decorations if the "disable shroud" cheat is activated.